HHS/OPRE-OPS
# Minor FakeAuth® tweak

## What changed

As discussed recently, the recently-added budget team users's (#2653) OIDC ID needs to get included into the default list of FakeAuth® user ID's to fully get all the rights, privileges, and exceptions being a FakeAuth® user affords such a user. 

Additionally, also out of some discussion and frustrating test experiences, the source IP check was removed for now (#2739) so I'm removing an errant reference to that in a relevant user function that I saw while looking at the other thing all in the name of _"Leave code cleaner than you found it"_ from our TWA.

## Issue

none, just casual usage and everyday QA.

## How to test

Log in as the budget team user, let your session persist beyond 30 minutes and see what happens... you should stay logged in.
